jupytercalpoly/Jupyter-Bifrost
# Variables Tab

**Added basic pill functionality to the Variables Tab**

- Added Pill component.
- Added css-in-js themes.
- Added the ability to add and delete encodings.

## Testing
1. Open up the Bifrost Widget in the `Introduction.py` notebook.
2. Try clicking the proceed button without selecting columns. Verify that no action is taken.
3. Choose columns and go to next screen. Choose graph.
4. Click on an encoding pill and select a new column. Verify that graph changes.
5. Click on encoding pill twice. Verify that color selection toggles.
6. Click on the X on the pill. Verify that the pill is deleted and the graph changes.
7. Click on the plus icon.
8. Verify that you cannot add a duplicate encoding pill.
9. Verify that you can add a new encoding.
